indico_toolkit/snapshots/snapshot.py

```diff
@@ -214,6 +216,28 @@ def split_and_write_to_csv(
             rows_taken += split_length
             print(f"Wrote split {i} of {num_splits}: {split_file_loc}")
 
+    def update_snapshot_shape(self, task_type: str = "annotation"):
```

Comment:
update_snapshot_shape feels like a weird name for this because "df shape" means something very specifc (# cols, # rows)

Reply:
Changed name to `update_label_col_format` 

Resolved in commit [b06727e](https://github.com/IndicoDataSolutions/Indico-Solutions-Toolkit/pull/138/commits/b06727efc5c16e054f9c2132fe820897c797f58a)